tools/bridge-deploy/bridge_deploy/cli.py

```diff
@@ -27,6 +30,16 @@ def validate_address(ctx, param, value):
         ) from e
 
 
+def validate_percentage_value(ctx, param, value):
+    if value >= 0 and value <= 100:
+        return value
+
+    else:
+        raise click.BadParameter(
+            f"The percentage value is not in the valid range of 0 till 100"
```

Comment:
```suggestion
            f"The percentage value is not in the valid range of 0 to 100"
```

Reply:
I was searching for a word that describes that 100 is also included. So to seem to be better. :+1:

Reply:
:heavy_check_mark: